content/en/docs/concepts/certificate.md

```diff
@@ -62,6 +62,39 @@ which contains the following explanation:
 > assumption that the remote end must already possess it in order to
 > validate it in any case.
 
+{{% alert title="Use ca.crt with caution" color="warning" %}}
```

Comment:
suggestion: I like the general idea of this, but I think the two sections are so similar that they could be condensed down into one shorter block. I can see the reason for having two - mutual-TLS isn't symmetric - but I think in this case the concepts are simple enough to have an example for configuring a client to trust a server's cert, and then a line underneath with something like:

"The same concept also applies when configuring a server for mutually-authenticated TLS; don't use an actual Secret containing a client certificate."